examples/Watchdog_live_view_demo.ipynb

```diff
@@ -0,0 +1,489 @@
+    "            print(\"All files produced, stopping monitoring...\")\n",
+    "            self.observer.stop()\n",
+    "            self.stop_flag[0] = True"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The function below initialises the Watchdog Observer to monitor the desired directory.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def main(working_directory=os.getcwd()):\n",
+    "    \"\"\"\n",
+    "    Function to plot hv plots whilst simultaneously running a Time Driver simulation\n",
+    "\n",
+    "    Parameters:\n",
+    "    working_directory: The directory we want Watchdog to monitor. Defaulted to the current working directory\n",
+    "    \"\"\"\n",
+    "\n",
+    "    # stop flag used to terminate thread when all files are produced.\n",
+    "    stop_flag = [False]\n",
+    "\n",
+    "    # Set up the observer\n",
+    "    observer = Observer()\n",
+    "    handler = DirectoryMonitorHandler(observer, stop_flag)\n",
+    "\n",
+    "    # Schedule the observer to monitor the working directory\n",
+    "    observer.schedule(handler, working_directory, recursive=True)\n",
+    "    print(f\"Monitoring: {working_directory}\")\n",
+    "\n",
+    "    # Start the observer\n",
+    "    observer.start()\n",
+    "\n",
+    "    try:\n",
+    "        while not stop_flag[0]:\n",
+    "            time.sleep(5)  # Keeps the script running\n",
+    "    except KeyboardInterrupt:\n",
+    "        print(\"Stopping monitoring...\")\n",
+    "    finally:\n",
+    "        observer.stop()\n",
+    "\n",
+    "    observer.join()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Run the cell below first to begin directory monitoring and then run the Time Driver simulation in your other notebook.\n",
+    "\n",
+    "\n",
+    "**Remember** to specify path of the directory you want monitored if your simulation files aren't being produced in the current working directory."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Monitoring: C:\\Users\\Noahd\\IP code\n",
+      "New directory created: C:\\Users\\Noahd\\IP code\\test_1\\drive-104\n",
+      "Now monitoring: C:\\Users\\Noahd\\IP code\\test_1\\drive-104\n",
+      "New file detected: C:\\Users\\Noahd\\IP code\\test_1\\drive-104\\m0.omf\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "8437b4eb022c41d39e9370ca406c299b",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "BokehModel(combine_events=True, render_bundle={'docs_json': {'395b53fb-3f0c-4419-a8ef-eb0a553b443d': {'version…"
+      ]
+     },
+     "metadata": {},
```

Comment:
We should get rid of the host-specific information. Maybe add an option `verbose=False` and only print when calling `main(verbose=True)`.